src/libServer/WebsocketServer.cpp

```diff
@@ -260,10 +282,47 @@ void WebsocketServer::on_message(const connection_hdl& hdl,
         }
         break;
       }
+      case TXNLOG: {
+        set<Address> track_addresses;
+        if (j_query.isMember("addresses") && j_query["addresses"].isArray() &&
+            !j_query["addresses"].empty()) {
+          for (const auto& addr_str : j_query["addresses"]) {
+            try {
+              if (!JSONConversion::checkStringAddress(addr_str.asString())) {
```

Comment:
Actually if the address is not valid, it will be caught in the try catch because `Address addr(addr_str.asString())` throw error, so maybe the `checkStringAddress` here is not that necessary?

Reply:
It actually does not check the address size (for eg, 0x11 is a valid address for conversion)

Reply:
Oh, in that case would you mind to help to apply that to EVENTLOG as well? Thank you!

Reply:
hey the EVENTLOG code already has  
```
if (acc == nullptr || !acc->isContract()) {
```